### PR TITLE
Sys info lang

### LINKED ISSUE #10410: Use unlocalized (english) values for system information
Nothing important, just something on my wishlist:

With support mails coming in from different countries, I have seen, that the system information uses some localized strings, e.g. for a french user:

` geocaching.com: Logged in (Connect%3%9) / BASIC`
` BASE: /cgeo (D%3%9fini par l'utilisateur)[/...`
`Routing mode: %3� pied`

I have some hard time to decipher those (not only due to special chars, but also in some languages I simply don't understand).

If its possible I would prefer always to see the english strings here (if technically possible).

### PR DESCRIPTION
**Pull request by fm-sys**
- Use unlocalized (english) values
- Add hint that value is localized (at places where an english value isn't easily possible)
- Avoid html styling tags for attribution text

fix #10410

### CODE REVIEW

File: main/src/cgeo/geocaching/utils/SystemInformation.java [hunk 1]
```diff
@@ -188,16 +188,17 @@ private static void appendPersistedDocumentUris(@NonNull final StringBuilder bod
...
         if (source == null) {
             body.append("none");
             return;
         }
         final ImmutablePair<String, Boolean> mapAtts = source.calculateMapAttribution(ctx);
-        body.append(source.getName()).append("\n  - Id: ").append(source.getId())
-            .append("\n  - Atts: ").append(mapAtts == null ? "none" : mapAtts.left)
-            .append("\n  - Theme: ").append(Settings.getSelectedMapRenderTheme());
+        body.append(source.getName()) // unfortunately localized but an English string would require large refactoring. The sourceId provides an unlocalized and unique identifier.
+                .append("\n  - Id: ").append(source.getId())
```
Comment (Lineflyer) [inline main/src/cgeo/geocaching/utils/SystemInformation.java:199]: That should be no problem to have it not localized.

File: main/src/cgeo/geocaching/utils/SystemInformation.java [hunk 2]
```diff
@@ -219,7 +220,7 @@ private static void appendConnectors(@NonNull final StringBuilder body) {
                 if (connector instanceof ILogin) {
                     final ILogin login = (ILogin) connector;
                     connectors.append(": ").append(login.isLoggedIn() ? "Logged in" : "Not logged in")
-                            .append(" (").append(login.getLoginStatusString()).append(')');
+                            .append(" (localized message: ").append(login.getLoginStatusString() /* unfortunately localized but an English string would require large refactoring */).append(')');
                     if (login.getName().equals("geocaching.com") && login.isLoggedIn()) {
```
Comment (Lineflyer) [inline main/src/cgeo/geocaching/utils/SystemInformation.java:224]: missing "not" (I guess)

File: main/src/cgeo/geocaching/utils/SystemInformation.java [hunk 3]
```diff
@@ -219,7 +220,7 @@ private static void appendConnectors(@NonNull final StringBuilder body) {
                 if (connector instanceof ILogin) {
                     final ILogin login = (ILogin) connector;
                     connectors.append(": ").append(login.isLoggedIn() ? "Logged in" : "Not logged in")
-                            .append(" (").append(login.getLoginStatusString()).append(')');
+                            .append(" (localized message: ").append(login.getLoginStatusString() /* unfortunately localized but an English string would require large refactoring */).append(')');
```
Comment (Lineflyer) [inline main/src/cgeo/geocaching/utils/SystemInformation.java:223]: I think its not necessary here to prepend "Localized message" as it is easy to see

File: main/src/cgeo/geocaching/utils/SystemInformation.java [hunk 4]
```diff
@@ -188,16 +188,17 @@ private static void appendPersistedDocumentUris(@NonNull final StringBuilder bod
     }
 
     private static void appendMapSourceInformation(@NonNull final StringBuilder body, @NonNull final Context ctx) {
-        body.append("\n- Map: ");
+        body.append("\n- Map (localized name): ");
```
Comment (Lineflyer) [inline main/src/cgeo/geocaching/utils/SystemInformation.java:191]: I think its not needed to prepend "Localized name", its easy to see

**Comment by fm-sys** [comment]
Ok, I removed the "Localized name" prefix

**Comment by moving-bits** [comment]
Failing tests are due to the website change only, so let's merge.

**Pull request merged**